engine/userguide/eng-image/dockerfile_best-practices.md

```diff
@@ -283,6 +283,19 @@ keep the image size down. Since the `RUN` statement starts with
 > **Note**: The official Debian and Ubuntu images [automatically run `apt-get clean`](https://github.com/docker/docker/blob/03e2923e42446dbb830c654d0eec323a0b4ef02a/contrib/mkimage/debootstrap#L82-L105),
 > so explicit invocation is not required.
 
+#### Using pipes
+
+Where your `RUN` command depends on pipes, for example:
+
+
+	RUN wget -O - https://some.site | wc -l > /number
+
+Remember that Docker executes this with `/bin/sh -c` which considers the command successful if the last operation in the pipe is successful. In our example above this build step will succeed and result in a new image produced so long as `wc -l` succeeds, even if the `wget` fails.
```

Comment:
s/Remember that Docker executes this...successful./Docker executes these commands using the `/bin/sh -c` interpreter, which only evaluates the exit code of the last operation in the pipe to determine success.

s/In our example above/In the example above

s/will succeed and result/succeeds and produces a new image

s/the `wc -l` succeeds/the `wc -l` command succeeds (and the same for `wget`)